deeppavlov/core/commands/train.py

```diff
@@ -127,21 +141,9 @@ def train_model_from_config(config_path: str) -> None:
     iterator: Union[DataLearningIterator, DataFittingIterator] = from_params(iterator_config,
                                                                              data=data)
 
-    if 'chainer' in config:
-        model = fit_chainer(config, iterator)
-    else:
-        vocabs = config.get('vocabs', {})
-        for vocab_param_name, vocab_config in vocabs.items():
-            v: Estimator = from_params(vocab_config, mode='train')
-            vocabs[vocab_param_name] = _fit(v, iterator, None)
-
-        model_config = config['model']
-        model = from_params(model_config, vocabs=vocabs, mode='train')
-
     train_config = {
         'metrics': ['accuracy'],
-
-        'validate_best': True,
+        'validate_best': not (is_trained),
```

Comment:
I would leave it `True` by default, because it checks if the model saves and loads correctly.